apps/asap-server/src/handlers/teams/users-handler.ts

```diff
@@ -0,0 +1,77 @@
+import { EventBridgeEvent } from 'aws-lambda';
+import { ListResponse, UserResponse } from '@asap-hub/model';
+import { SquidexGraphql } from '@asap-hub/squidex';
+import {
+  AlgoliaSearchClient,
+  algoliaSearchClientFactory,
+  BatchRequest,
+} from '@asap-hub/algolia';
+import Users, { UserController } from '../../controllers/users';
+import {
+  TeamsEventType,
+  SquidexWebhookTeamPayload,
+} from '../webhooks/webhook-teams';
+import { loopOverCustomCollection } from '../../utils/migrations';
+import logger from '../../utils/logger';
+import { algoliaApiKey, algoliaAppId, algoliaIndex } from '../../config';
+
+export const indexTeamsUsersHandler =
+  (
+    userController: UserController,
+    algoliaClient: AlgoliaSearchClient,
+  ): ((
+    event: EventBridgeEvent<TeamsEventType, SquidexWebhookTeamPayload>,
+  ) => Promise<void>) =>
+  async (
+    event: EventBridgeEvent<TeamsEventType, SquidexWebhookTeamPayload>,
+  ): Promise<void> => {
+    logger.debug(`Event ${event['detail-type']}`);
+
+    try {
+      const fetchFunction = (
```

Comment:
actually for this handler I dont think you need to fetch-users-by-team-relationship. Why don't use simply fetch the team by ID via `TeamController` and use the `members` list to update the users?